### ISSUE TITLE
Rewrite tests that use infura requests

### ISSUE THREAD

**Issue by igor-sirotin** [status-im/status-go#5087]
We shouln't make infura requests during status-go tests.
For now they're skipped: https://github.com/status-im/status-go/pull/5086

**Comment by supragya** [comment]
Shouldn't this be closed? Since #5086 is merged? @igor-sirotin

**Comment by igor-sirotin** [comment]
> Shouldn't this be closed? Since #5086 is merged? @igor-sirotin 

Not really. #5086 was a hotfix to make tests working again, i.e. just skip them. We should rewrite those tests in a proper way, this is what the issue is about now.
